src/ontology/uberon.Makefile

```diff
@@ -98,250 +120,43 @@ $(OWLSRC): $(SRC) $(COMPONENTSDIR)/disjoint_union_over.ofn $(REPORTDIR)/$(SRC)-g
-
-## MERGED UNREASONED ONTOLOGY
-##
-## TODO - restore Disjoints. _ OLD NOTE.  DELETE?
-## TODO - get rid of declarations and inferred subclass axioms for other ontology classes _OLD NOTE. DELETE?
-
-# TODO - Make uberon-bridge-to-bfo.owl a component.  BUT it needs to stay in the bridges directory!
-$(TMPDIR)/unreasoned.owl: $(OWLSRC) $(BRIDGEDIR)/uberon-bridge-to-bfo.owl # $(COMPONENTSDIR)/phenoscape-ext.owl
-	$(OWLTOOLS) $^ --merge-support-ontologies -o -f functional $@
-
-# First pass at making base module
-# Currently this will be missing the temporary reflexivity axioms.
-# should this include downward-injected axioms, e.g on ZFA?
-#uberon-base.owl: $(TMPDIR)/unreasoned.owl
-#	$(OWLTOOLS) $< --remove-imports-declarations --remove-axioms -t Declaration --set-ontology-id -v $(RELEASE)/$@ $(URIBASE)/uberon/$@ -o $@.tmp && mv $@.tmp $@
-
-# TODO document collected-* pattern (import files see http://uberon.github.io/downloads.html#multiont)
-# ----------------------------------------
-# STEP 3: Perform reasoning and create release ext.owl file
-# ----------------------------------------
-
-# ext.owl is the release file that includes full imports and inter-ontology axioms
-#ext.owl: $(TMPDIR)/materialized.owl
-#	$(ROBOT) reason -i $< -r elk relax reduce -r elk annotate --ontology-iri $(URIBASE)/uberon/$@ -V  $(RELEASE)/$@ -o $@
-
-## TESTING - NEW
-$(TMPDIR)/is_ok: $(TMPDIR)/materialized.owl
-	$(OWLTOOLS) $< --run-reasoner -r elk -u > $@.tmp && mv $@.tmp $@
-
-# somewhat awkward: we temporarily inject reflexivity axioms
-# **Hacking_Feb_2022** Notes start here
-# Background: 
-## This ends up in the release file, reflexivity_axioms.owl temp merge as consistency QC
-## This is only a QC.  Rationale - this would catch some errors because expands part disjointness ()
-## **Hacking_Feb_2022** TODO - rewrite as as expansions from annotation axioms using SPARQL. Expanded axioms --> component (as for taxon restrictions).
-
-TMP_REFL=$(COMPONENTSDIR)/reflexivity_axioms.owl
+# Step 2: Reasoning.
+# For the temporary injection of property chains, see
+# <https://github.com/obophenotype/uberon/issues/2381>
 DEVELOPS_FROM_CHAIN=$(COMPONENTSDIR)/develops-from-chains.owl
-# see https://github.com/obophenotype/uberon/issues/2381
-
-$(TMPDIR)/materialized.owl: $(TMPDIR)/unreasoned.owl $(TMP_REFL)
-	$(ROBOT) merge -i $< -i $(DEVELOPS_FROM_CHAIN) --collapse-import-closure false \
-		relax \
-		materialize -T $(CONFIGDIR)/basic_properties.txt -r elk \
-		reason -r elk --exclude-duplicate-axioms true --equivalent-classes-allowed asserted-only \
-		unmerge -i $(TMP_REFL) \
-		unmerge -i $(DEVELOPS_FROM_CHAIN) \
-		annotate -O $(URIBASE)/uberon/materialized.owl -V  $(RELEASE)/materialized.owl -o $@ 2>&1 > $@.LOG
-.PRECIOUS: $(TMPDIR)/materialized.owl
-
-# Used directly by Bgee, see https://github.com/obophenotype/uberon/issues/1501
-ext.owl: $(TMPDIR)/materialized.owl
```

Comment:
Given the comment above "used directly by bgee", I would prefer to change this goal to 

```
ext.owl: uberon.owl
	$(ROBOT) annotate -i $< --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@
```

instead of deleting it. Remember users have idiosyncratic pipelines, and they may have lost over time developers etc that can debug running pipelines -> if a pipeline breaks because we remove a (I agree, redundant) release artefact, this can cause churn and reduce trust. I think its a tiny compromise to leave it in as a copy of uberon.owl.

Reply:
I would rather recommend updating Uberon’s PURL configuration to redirect `http://purl.obolibrary.org/obo/ext.owl` to `uberon.owl`.

We have a URL redirection system, we might as well use it.

Reply:
This is an option, albeit a little bit less safe in case a pipeline would use github links instead of PURLs.. remember my position here is not to challenge the correctness, more to reduce churn for users - even those using our resources "the wrong way".

Reply:
I strongly object to keeping cruft in the Makefile just because _maybe_, _somewhere_, it could be that _someone_ _may_ be relying on that cruft – not if I don’t have the slightest clue to tell me that it is indeed the case.

If this breaks someone’s pipeline, they can complain about it. Then they can explain to use why they use direct GitHub URLs, and if their explanation makes sense, we can add back the `ext` product.

Using direct GitHub URLs always exposes users to unnecessary trouble and it should be clearly stated that this is unsupported.

(This applies to us: I am not particularly happy about the fact that we clone the `developmental-stages-ontologies` repo as part of this Makefile; It would be much better if that repo made some proper releases that we could then just download as any other foreign ontology.)

Reply:
When I said below that one aim of this PR was “not changing the current behaviour”, I was speaking of the officially released products published in the official way.

If we have a PURL redirection from `ext` to `uberon`, the officially released `ext` product would appear to anyone out there to be the same as it was before (except for its ontology IRI), so this wouldn’t change the “current behaviour”.

Reply:
> not if I don’t have the slightest clue to tell me that it is indeed the case.

The phrase "Used directly by Bgee, see https://github.com/obophenotype/uberon/issues/1501" should at least approximate your definition of "slightest clue". But let's not ride on it. This is a sort of 51/49 situation for me now, so since you feel reasonably strongly about it, I am inclined to just drop my objections.

Reply:
resolving in favour of https://github.com/obophenotype/uberon/issues/3031